server/cluster/store_limiter.go

```diff
@@ -47,42 +44,6 @@ func NewStoreLimiter(opt *config.PersistOptions) *StoreLimiter {
 	}
 }
 
-// Collect the store statistics and update the cluster state
-func (s *StoreLimiter) Collect(stats *pdpb.StoreStats) {
-	s.m.Lock()
-	defer s.m.Unlock()
-
-	log.Debug("collected statistics", zap.Reflect("stats", stats))
-	s.state.Collect((*StatEntry)(stats))
-
-	state := s.state.State()
-	ratePeerAdd := s.calculateRate(storelimit.AddPeer, state)
-	ratePeerRemove := s.calculateRate(storelimit.RemovePeer, state)
-
-	if ratePeerAdd > 0 || ratePeerRemove > 0 {
-		if ratePeerAdd > 0 {
-			s.opt.SetAllStoresLimit(storelimit.AddPeer, ratePeerAdd)
-			log.Info("change store region add limit for cluster", zap.Stringer("state", state), zap.Float64("rate", ratePeerAdd))
-		}
-		if ratePeerRemove > 0 {
-			s.opt.SetAllStoresLimit(storelimit.RemovePeer, ratePeerRemove)
-			log.Info("change store region remove limit for cluster", zap.Stringer("state", state), zap.Float64("rate", ratePeerRemove))
-		}
-		s.current = state
-		collectClusterStateCurrent(state)
-	}
-}
-
-func collectClusterStateCurrent(state LoadState) {
-	for i := LoadStateNone; i <= LoadStateHigh; i++ {
-		if i == state {
-			clusterStateCurrent.WithLabelValues(state.String()).Set(1)
-			continue
-		}
-		clusterStateCurrent.WithLabelValues(i.String()).Set(0)
-	}
-}
-
 func (s *StoreLimiter) calculateRate(limitType storelimit.Type, state LoadState) float64 {
```

Comment:
remove it?

Reply:
yes, there are no code to depence it.

Reply:
BTW, /stores/limit/scene should be removed since we have no any other scene